crates/sncast/src/helpers/interactive.rs

```diff
@@ -76,22 +76,19 @@ fn edit_config(config_path: &Utf8PathBuf, profile: &str, key: &str, value: &str)
 }
 
 fn update_config(toml_doc: &mut DocumentMut, profile: &str, key: &str, value: &str) {
-    if !toml_doc.contains_key("sncast") {
-        toml_doc["sncast"] = Item::Table(Table::new());
-    }
+    let sncast_secetion = toml_doc
```

Comment:
How did it pass the `typos` check? 🤔 
```suggestion
    let sncast_section = toml_doc
```

Reply:
I don't know how it works, but maybe words with underscores aren't checked.